packages/unmock-core/src/interfaces.ts

```diff
@@ -66,3 +66,23 @@ export interface IUnmockOptions {
   useInProduction?: boolean;
   mode?: Mode;
 }
+
+export interface ISerializedRequest {
```

Comment:
Do we want to leave the possibility open for protocol?

Reply:
Added protocol.